client/views/login/login.html

```diff
@@ -13,9 +13,12 @@ <h1 class="formtitle">Login</h1>
 			<div class="modalbottommessage">
 				Don't have an account? <p class="modalemphasis" id="registeremphasis">Register</p>
 			</div>
+			<div class="modalbottommessage">
+				Forgot password? <p class="modalemphasis" id="forgotpasswordemphasis">Send instruction</p>
```

Comment:
@singhsanket143 `Send instruction` > `Send instructions`

Reply:
@peter-hank I have updated the text here